### ISSUE TITLE
AAPT Deprecation

### ISSUE THREAD

**Issue by rye** [StoDevX/AAO-React-Native#2629]
From a local build:

```
WARNING: AAPT is deprecated and support for it will be soon removed. [wrap]
Remove the 'android.enableAapt2=false' flag to enable AAPT2.
```

**Comment by hawkrives** [comment]
There may be another issue / PR / chat with the information about why I disabled that. 

IIRC something something RN's asset pipeline didn't work with aapt2.

**Comment by rye** [comment]
We should keep an eye on this, then.  Eventually it'll be the ~default~ only option?  Or something like that.

**Comment by hawkrives** [comment]
Yup.

**Comment by drewvolz** [comment]
Fixed in the upgrade to 0.59
